src/runtime/virtcontainers/hypervisor.go

```diff
@@ -370,6 +370,9 @@ type HypervisorConfig struct {
 	// VhostUserStorePathList is the list of valid values for vhost-user paths
 	VhostUserStorePathList []string
 
+	// SeccompSandbox is the qemu function which enables the seccomp feature
+	SeccompSandbox string
+
```

Comment:
@fengwang666 I'm wondering if `SeccompSandbox` has any interactions with other members of `HypervisorConfig`. If so maybe [`Valid()`](https://github.com/kata-containers/kata-containers/blob/185360cb9a194ed2b08d31488c842d7d4ed24cd6/src/runtime/virtcontainers/hypervisor.go#L568) should be extended accordingly?

Reply:
I don't *think* it does? Either way, it is QEMU specific so I don't think it should really go into Valid()

Reply:
Fair enough, just making sure we considered this.